src/utils/utils.js

```diff
@@ -639,6 +639,11 @@ function getDefinedTraits(message) {
       message?.context?.traits?.lastname ||
       message?.context?.traits?.last_name,
     name: message?.context?.traits?.name || message?.context?.traits?.Name,
+    state:
+      message?.context?.traits?.state ||
+      message?.context?.traits?.State ||
+      message?.context?.traits?.address?.state ||
+      message?.context?.traits?.address?.State,
```

Comment:
Looks like a breaking change. Have we checked other destinations which are using this function?

Reply:
Checked for Clevertap.

```
rudderanalytics.identify("userId123", {
          email: "clevertap_test@email.com",
          phone: "8123099213",
          name: "Clevertap Test",
          state: "UP",
          country: "IN"
        });
```
Values are coming perfectly!
<img width="1728" alt="Screenshot 2022-12-26 at 1 58 15 PM" src="https://user-images.githubusercontent.com/60897972/209525094-162fef31-6e9c-49a6-8a19-39bd461c6095.png">